### PR TITLE
[T4A3][W13-B2]YongLinHan

### PR DESCRIPTION
**Pull request by linhany**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/person/Contact.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's contacts in the address book.
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:4]: this is good, but try to avoid specifics in the header comment. make it as generic as possible.

File: src/seedu/addressbook/data/person/Contact.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ */
+public class Contact {
+    protected String value;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:7]: try to make it private. in general, start off with private members, then use protected or public only if absolutely necessary.

File: src/seedu/addressbook/data/person/Contact.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    
+    @Override
+    public int hashCode() {
+        return value.hashCode();
+    }
+    
+    public boolean isPrivate() {
+        return isPrivate;
+    }
+}
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Contact.java:23]: - i see that you haven't created a constructor in the parent class; you could use a `super()` call to set the members, no?
- think about how you can push more stuff to the parent class in this case and in general.

File: src/seedu/addressbook/storage/jaxb/AdaptedPerson.java [hunk 4]
```diff
@@ -52,15 +52,15 @@ public AdaptedPerson(ReadOnlyPerson source) {
...
 
         email = new AdaptedContactDetail();
         email.isPrivate = source.getEmail().isPrivate();
-        email.value = source.getEmail().value;
+        email.value = source.getEmail().toString();
 
         address = new AdaptedContactDetail();
         address.isPrivate = source.getAddress().isPrivate();
-        address.value = source.getAddress().value;
+        address.value = source.getAddress().toString();
```
Comment (okkhoy) [inline src/seedu/addressbook/storage/jaxb/AdaptedPerson.java:63]: this commit doesn't seem to be related to this activity. try to keep your PR clean.

**Comment by okkhoy** [comment]
@ylhlh 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/storage/jaxb/AdaptedPerson.java (hunk 4 above)
Comment (linhany) [reply src/seedu/addressbook/storage/jaxb/AdaptedPerson.java:63]: because the values are no longer public, have to change this to make the program work
